deployments/icesat2/config/staging.yaml

```diff
@@ -12,6 +12,33 @@ pangeo:
           memory: 1G
       extraEnv:
         OAUTH_CALLBACK_URL: "https://staging.aws-uswest2.pangeo.io/hub/oauth_callback"
+    singleuser:
+      profileList:
+        - display_name: "Staging Pangeo-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/tree/master/pangeo-notebook"
+          kubespawner_override:
+            image: pangeo/pangeo-notebook:master
+        - display_name: "Latest Pangeo-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/releases"
+          kubespawner_override:
+            image: pangeo/pangeo-notebook:latest
+        - display_name: "Staging Base-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/tree/master/base-notebook"
+          kubespawner_override:
+            image: pangeo/base-notebook:master
+        - display_name: "Staging ML-notebook"
+          description: "https://github.com/pangeo-data/pangeo-docker-images/tree/master/ml-notebook"
```

Comment:
I think title or description should indicate you get a GPU machine, as that influence how the user may want to manually shut down the pod or so to save some money.

Reply:
good call. Maybe "ML Notebook with GPU, please only use if you need it ;)"